src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,76 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String DEFAULT_COLOR = "White";
+    private static final ColorSupplier COLOR_SUPPLIER = new ColorSupplier();
+    private static final Random RANDOM = new Random();
+    private int randomNumber = RANDOM.nextInt(5);
+    private double figuresSidesNumber = RANDOM.nextInt(100);

```

Comment:
numbers are 100 or 5 and also 10.0. These are the ones that need to be made constants. Here is an example
```suggestion
    private static final int MAX_RANDOM_NUMBER = 100;
```